src/site/_filters/i18n.js

```diff
@@ -74,9 +74,11 @@ const i18n = (path, locale = defaultLocale) => {
       return out;
     }
   } catch (err) {
-    // ignore, throw below
+    // ignore, return an empty string in the case of no i18n result.
   }
-  throw new Error(`Could not find i18n result for: ${path}`);
+
+  console.warn('Could not find i18n result');
```

Comment:
I see why that change is necessary, but it's not ideal... building the site no prints 816x 'Could not find i18n result', which might be confusing for users... also not throwing anymore can easily introduce translation errors, which would only make the build process render the warning 817 times 😄

The new i18n implementation in webdev-infra falls back to the `en` translation instead, and only errors if that doesn't exist too. I think we should do the same here as an interim solution once we switch to the new i18n filter.